src/main/java/com/cinema/validation/EmailValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.cinema.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<EmailConstraint, String> {
+    public static final String VALID_EMAIL = "^(.+)@(.+)$";
+
+    @Override
+    public void initialize(EmailConstraint constraintAnnotation) {
+    }
```

Comment:
I think this chunk of code can be removed

Reply:
> I think this chunk of code can be removed

agree